net-voip/sipvicious/sipvicious-0.3.4.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1
+
+DESCRIPTION="SIPVicious is a SIP security package"
+HOMEPAGE="https://github.com/EnableSecurity/sipvicious/wiki"
+SRC_URI="https://github.com/EnableSecurity/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Could plain-text this ${PN}, see:
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D_in_SRC_URI

Reply:
And plaintexting this makes additional requirement for change on version to version ... so how do you balance these conflicting ideals?  emerge -pf is useful and easy for the case described given in the referenced link, even if not plaintexted here.

Reply:
This was only about ${PN}, when does ${PN} change?